locales/en.yml

```diff
@@ -181,6 +181,16 @@ en:
       management_network_required: |-
         Management network can't be disabled when VM use box.
         Please fix your configuration and run vagrant again.
+      snapshot_deletion_error: |-
+        Error while deleting snapshot: %{error_message}.
```

Comment:
So, I'm getting an InterpolationError here because of this variable. Everything looks corret to me... What might be causing that?

Reply:
When raising the exception, need to pass something as the `error_message` value, see how the following is raised:
https://github.com/vagrant-libvirt/vagrant-libvirt/blob/1b8a70163bef05759c42bb7912810c6d8fc22c19/lib/vagrant-libvirt/action/create_network_interfaces.rb#L33-L36

Reply:
I thought it was something simple like forgetting to pass it, but it's not. Still looking.

Reply:
I will try to take a look into that this week when I have some time. I tried to look it up when I opened the PR, but I completely forgot about this... If you find out why let me know and I can fix it as well.

Reply:
Essentially, the problem on this one was because I was not passing the variable `error_message` when raising the `Errors::SnapshotMissing` class

Reply:
I'm resolving this conversation since it was me not paying enough attention to the code.